css-view-transitions-2/Overview.bs

```diff
@@ -523,6 +520,8 @@ The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
 		1. Set |oldDocument|'s [=active view transition=] to |outboundTransition|.
 
 			Note: The process continues in [=setup view transition=], via [=perform pending transition operations=].
+
+		1. The user agent should display the currently displayed frame until |newDocument| is [=revealed=], or until its [=active view transition=]'s [=ViewTransition/phase=] is "`done`".
```

Comment:
Trying to reason about why we need "or until its [=active view transition=]'s [=ViewTransition/phase=] is "`done`". Is this for the case where the transition is skipped before reveal? I think that can only happen in the reveal algorithm [here](https://drafts.csswg.org/css-view-transitions-2/#document-reveal:~:text=Let%20transition%20be%20the%20result%20of%20activating%20cross%2Ddocument%20view%2Dtransition%20for%20document.) anyway which is right before we hit "revealed".

Reply:
If the document navigates away or an SPA nav is initiated.

Reply:
Ah ok cool. Slight suggestion on wording:

The user agent should display the currently displayed frame until either:
- |newDocument| is [=revealed=]
- its [=active view transition=]'s [=ViewTransition/phase=] is "`done`"

Non-normative note to document the cases you mentioned for the second condition.

Reply:
LGTM % this comment.

Reply:
Done